ledger/sandbox-common/src/test/lib/scala/platform/sandbox/SandboxRequiringAuthentication.scala

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox
+
+import java.time.{Duration, Instant}
+import java.util.UUID
+
+import com.daml.jwt.domain.DecodedJwt
+import com.daml.jwt.{HMAC256Verifier, JwtSigner}
+import com.daml.ledger.api.auth.{
+  AuthService,
+  AuthServiceJWT,
+  AuthServiceJWTCodec,
+  AuthServiceJWTPayload
+}
+import com.daml.ledger.api.domain.LedgerId
+import org.scalatest.Suite
+import scalaz.syntax.tag.ToTagOps
+
+trait SandboxRequiringAuthentication {
```

Comment:
```suggestion
trait SandboxRequiringAuthorization {
```
And rename the file, please.

Reply:
Sold!